ksqldb-engine/src/test/java/io/confluent/ksql/schema/ksql/inference/SchemaRegistryTopicSchemaSupplierTest.java

```diff
@@ -535,11 +537,17 @@ public void shouldRequestCorrectSchemaOnGetKeySchemaWithId() throws Exception {
 
   @Test
   public void shouldPassRightSchemaToFormat() {
+    // Given:
+    Map<String, String> properties = new ImmutableMap.Builder<String, String>()
+        .putAll(formatProperties)
+        .put("VALUE_SCHEMA_ID", "12")
```

Comment:
What is the purpose of adding this in this unit test? Is this used in the test?

Reply:
Ah I see. I'd rename `properties` to `expectedProperties`. Either moving it to be right before line 550 or leaving it here is fine (I've seen both in the codebase).

My confusion was that I saw this was in the `Given` and thought it was being passed in as part of the `When` and then immediately checked again in the `Then`, but I now realize I misread. Clarifying its usage by renaming would've prevented my misread.